packages/pamgen/test/rtc_test/rtcompiler-test.cc

```diff
@@ -267,7 +267,7 @@ int main( int argc, char* argv[] )
   realProgram(arrayOneReal, arrayTwoReal, plainValOneReal,
               plainValTwoReal, plainValThreeReal, testChar);
 
-  const double epsilon = 1.0e-15;
+  [[maybe_unused]] const double epsilon = 1.0e-15;
```

Comment:
I didn't know about this.

Reply:
Let's see if that compiles everywhere...